configs/10.0/packages/binutils/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="GNU Binutils"
 ATSRC_PACKAGE_VER=2.27
-ATSRC_PACKAGE_REV=d72ca87af2af8
+ATSRC_PACKAGE_REV=b5d3ac25628b
```

Comment:
I see this revision has 11 characters instead of 12.
Wasn't we using 12 as a standard now?